civis/civis.py

```diff
@@ -13,6 +13,45 @@
 
 
 def find(object_list, filter_func=None, **kwargs):
+    """Return the objects from ``object_list`` that satisfy the filters.
+
+    Parameters
+    ----------
+    object_list : iterable
+        An iterable of arbitrary objects, particularly dicts and objects with
```

Comment:
This function won't filter dictionaries at all. It looks for attributes. That's one of the more confusing things about it, especially with the way the `Response` is written.